manticore/core/manticore.py

```diff
@@ -154,19 +72,11 @@ def __init__(self, path_or_state, argv=None, workspace_url=None, policy='random'
         # Link Executor events to default callbacks in manticore object
         self.forward_events_from(self._executor)
 
-        if isinstance(path_or_state, str):
```

Comment:
The `path_or_state` logic has been moved to the children class: `native/manticore.py:Manticore` since `make_initial_state` creates a target for a native state.